inappmessaging/USERGUIDE.md

```diff
@@ -387,6 +387,12 @@ Documents targeting Product Managers:
 
 ## <a name="changelog"></a> Changelog
 
+### NEXT (in progress)
```

Comment:
This should not be breaking change right?

Reply:
Not breaking change,  the host app can continue using the actual config.